templates/includes/related-tags.html

```diff
@@ -0,0 +1,69 @@
+    be shown conditionally when digital community content is displayed
+{% endcomment %}
+
+<div class="record-details__related-tags">
+    <h3>Related tags</h3>
+
+    <dl class="record-details__related-tags-list">
+        <dt>Location:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--location">
+                <span class="ohos-tag__inner">Location</span>
+
+                <a href="https://www.wikidata.org/wiki/Q208209" target="_blank" class="ohos-tag__link">
+                    Wikidata
+                    <i aria-hidden="true" class="ohos-tag__link-icon fa-solid fa-external-link"></i>
+                </a>
+            </span>
+        </dd>
+
+        <dt>Person:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--person">
+                <span class="ohos-tag__inner">Person</span>
+
+                <a href="https://www.wikidata.org/wiki/Q208209" target="_blank" class="ohos-tag__link">
+                    Wikidata
+                    <i aria-hidden="true" class="ohos-tag__link-icon fa-solid fa-external-link"></i>
+                </a>
+            </span>
+        </dd>
+
+        <dt>Organisation:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--organisation">
+                <span class="ohos-tag__inner">Organisation</span>
+
+                <a href="https://www.wikidata.org/wiki/Q208209" target="_blank" class="ohos-tag__link">
+                    Wikidata
+                    <i aria-hidden="true" class="ohos-tag__link-icon fa-solid fa-external-link"></i>
+                </a>
+            </span>
+        </dd>
+
+        <dt>Miscellaneous:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--misc">
+                <span class="ohos-tag__inner">Location</span>
+
+                <a href="https://test-deploy-ohos-xjy4lfq-rasrzs7pi6sd4.uk-1.platformsh.site/search/catalogue/?filter_keyword=&covering_date_from_0=&covering_date_from_1=&covering_date_from_2=&covering_date_to_0=&covering_date_to_1=&covering_date_to_2=&collection=ADM&opening_start_date_0=&opening_start_date_1=&opening_start_date_2=&opening_end_date_0=&opening_end_date_1=&opening_end_date_2=&q=&sort_by=&sort_order=asc&group=tna" class="ohos-tag__link">
+                    Collection
+                </a>
+            </span>
+        </dd>
+
+        <dt>Other:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--other">
+                <span class="ohos-tag__inner">Other</span>
+
+                <a href="https://test-deploy-ohos-xjy4lfq-rasrzs7pi6sd4.uk-1.platformsh.site/search/catalogue/?filter_keyword=&covering_date_from_0=&covering_date_from_1=&covering_date_from_2=&covering_date_to_0=&covering_date_to_1=&covering_date_to_2=&collection=ADM&opening_start_date_0=&opening_start_date_1=&opening_start_date_2=&opening_end_date_0=&opening_end_date_1=&opening_end_date_2=&q=&sort_by=&sort_order=asc&group=tna" class="ohos-tag__link">
```

Comment:
Do these need a `title` attribute? I'm just worried that the link just reads "Collection"

Reply:
Updated with a title attribute "See other (tag name) results in the collection". Once we have the data, tag name will be replaced with the actual name